sensors/sensors.go

```diff
@@ -22,16 +25,37 @@ type Lidar struct {
 	lidar camera.Camera
 }
 
-// TimedSensorReadingResponse represents a sensor reading with a time & allows the caller to know if the reading is from a replay sensor.
-type TimedSensorReadingResponse struct {
+// IMU represents an IMU movement sensor.
+type IMU struct {
+	Name string
+	imu  movementsensor.MovementSensor
+}
+
+// TimedLidarSensorReadingResponse represents a lidar sensor reading with a time &
+// allows the caller to know if the reading is from a replay camera sensor.
+type TimedLidarSensorReadingResponse struct {
 	Reading     []byte
 	ReadingTime time.Time
 	Replay      bool
 }
 
-// TimedSensor desribes a sensor that reports the time the reading is from & whether or not it is from a replay sensor.
-type TimedSensor interface {
-	TimedSensorReading(ctx context.Context) (TimedSensorReadingResponse, error)
+// TimedLidarSensor describes a sensor that reports the time the reading is from & whether or not it is from a replay sensor.
+type TimedLidarSensor interface {
+	TimedLidarSensorReading(ctx context.Context) (TimedLidarSensorReadingResponse, error)
+}
+
+// TimedIMUSensorReadingResponse represents an IMU sensor reading with a time & allows the caller to know if the reading is
+// from a replay movement sensor. Currently replay movement sensor are not yet supported.
+type TimedIMUSensorReadingResponse struct {
+	LinearAcceleration r3.Vector
+	AngularVelocity    spatialmath.AngularVelocity
+	ReadingTime        time.Time
+	Replay             bool
+}
+
+// TimedIMUSensor describes a sensor that reports the time the reading is from & whether or not it is from a replay sensor.
```

Comment:
[nit/opt] use and instead of &

Reply:
Copied comment structure over from lidar... want me to change both?